examples/storage/cassandra/README.md

```diff
@@ -393,21 +393,21 @@ NAME        DESIRED   CURRENT   AGE
 cassandra   4         4         36m
 ```
 
-For the Alpha release of Kubernetes 1.3 the Pet Set resource does not have `kubectl scale`
+For the Alpha release of Kubernetes 1.3 the StatefulSet resource does not have `kubectl scale`
```

Comment:
Should this updated to reflect that StatefulSet will be a beta feature in 1.5?
The scale limitation remains true.